### PR TITLE
Keep space between emit Qt keyword and opening parenthesis

### PR DESCRIPTION
**Pull request by MatthieuCOLLETTE**

### CODE REVIEW

File: lib/Format/TokenAnnotator.cpp [hunk 1]
```diff
@@ -49,6 +49,11 @@ static bool IsStdFunctionReturnType( FormatToken* token )
   return false;
 }
 
+static bool IsQtSignalEmit( FormatToken* token )
```
Comment (dguillois) [inline lib/Format/TokenAnnotator.cpp]: it is better if you can factorize the code in an helper file.

Comment (MatthieuCOLLETTE) [reply lib/Format/TokenAnnotator.cpp]: OK, I'll do this.

File: lib/Format/FormatHelper.cpp [hunk 2]
```diff
@@ -0,0 +1,37 @@
+#include "FormatHelper.h"
```
Comment (dguillois) [inline lib/Format/FormatHelper.cpp:16]: copyright is missing

File: lib/Format/FormatHelper.cpp [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  return false;
+}
+
+bool IsQtSignalEmit( FormatToken* token )
+{
+  return (token->is(tok::identifier) && token->TokenText == "emit");
+}
+
+}
+}
```
Comment (dguillois) [inline lib/Format/FormatHelper.cpp]: add a new line

File: lib/Format/FormatHelper.h [hunk 4]
```diff
@@ -0,0 +1,15 @@
+#ifndef LLVM_CLANG_LIB_FORMAT_HELPER_H
```
Comment (dguillois) [inline lib/Format/FormatHelper.h:16]: copyright is missing

File: lib/Format/FormatHelper.h [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+#define LLVM_CLANG_LIB_FORMAT_HELPER_H
+
+#include "FormatToken.h"
+
+namespace clang {
+namespace format {
+
+bool IsStdFunctionReturnType( FormatToken* token );
+bool IsQtSignalEmit( FormatToken* token );
```
Comment (dguillois) [inline lib/Format/FormatHelper.h]: token can be const ?

**Review by dguillois — CHANGES_REQUESTED** [review]

**Review by dguillois — APPROVED** [review]

**Pull request merged**